Dockerfile

```diff
@@ -6,14 +6,16 @@ COPY . .
 RUN yarn build
 
 FROM node:14-alpine AS clean-env
-COPY --from=build-env /sweater-comb/build /sweater-comb
+COPY --from=build-env /sweater-comb/build/ /sweater-comb/
+COPY --from=build-env /sweater-comb/babel.config.js /sweater-comb/
 COPY --from=build-env /sweater-comb/package*.json /sweater-comb/
 WORKDIR /sweater-comb
 RUN yarn install --production
 
-FROM gcr.io/distroless/nodejs:14
+FROM node:14-alpine
```

Comment:
This increases the image size from about 120MB to 180MB, but at least I can debug container issues...

Reply:
FWIW the reason I go to all this trouble.. the size of the build-env image is 600MB. That's just... no.